samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -820,10 +820,10 @@ Building with experimental support for Wi-Fi connectivity for nRF5340 DK with nR
 
```

Comment:
Can't quite hit the right line, but just noticed that the section title needs to drop the device reference as well.